staff/gianluca-bragaglia/cv-styled/index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>My CV</title>
+    <link href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/style.css">
+
+</head>
+<body>
+    <header>
+        <img src="./images/tiynyanov.png" alt="foto Jhon Dor">
+        <h1>John Dor</h1>
+        <h2>Web Developer</h2>
+        <address>
+            151 East 2º street <br> 
+            New York 12345 <br>
+            <a href="mailto:info@dor.com">info@dor.com</a>
+        </address>
+        <br>
+        <hr>
+    </header>
+    <br>
+    <div class="container">
```

Comment:
Maybe better use "MAIN" tan "DIV" for this section?

Reply:
I have 6 div container, I think is not good to put 6 main container

Reply:
fixed